### ISSUE TITLE
CEdit saves wrong fire with two fires in a compartment.

### ISSUE THREAD

**Issue by rpeacoc** [firemodels/cfast#1369]
Open and save the following test case and the constant fire is replaced with a copy of the t-squared fire.

```
&HEAD VERSION = 7500, TITLE = 'CFAST Simulation' /
 
!! Scenario Configuration 
&TIME SIMULATION = 3600 PRINT = 60 SMOKEVIEW = 15 SPREADSHEET = 15 / 
&INIT PRESSURE = 101325 RELATIVE_HUMIDITY = 50 INTERIOR_TEMPERATURE = 20 EXTERIOR_TEMPERATURE = 20 /
&MISC  LOWER_OXYGEN_LIMIT = 0.15 / 
 
!! Material Properties 
&MATL ID = 'GYPSUM' MATERIAL = 'Gypsum Board (5/8 in)', 
      CONDUCTIVITY = 0.16 DENSITY = 790 SPECIFIC_HEAT = 0.9, THICKNESS = 0.016 EMISSIVITY = 0.9 /
&MATL ID = 'Steel' MATERIAL = 'Steel', 
      CONDUCTIVITY = 48 DENSITY = 7854 SPECIFIC_HEAT = 0.559, THICKNESS = 0.0015 EMISSIVITY = 0.9 /
 
!! Compartments 
&COMP ID = 'Comp 1'
      DEPTH = 5 HEIGHT = 3 WIDTH = 5 CEILING_MATL_ID = 'GYPSUM' WALL_MATL_ID = 'GYPSUM' FLOOR_MATL_ID = 'OFF'
      ORIGIN = 0, 0, 0 GRID = 50, 50, 50 /
 
!! Wall Vents
&VENT TYPE = 'WALL' ID = 'WallVent_1' COMP_IDS = 'Comp 1' 'OUTSIDE'  TOP = 2, BOTTOM = 0, WIDTH = 1
  FACE = 'FRONT' OFFSET = 2 /
 
!! Fires 
&FIRE ID = 'Fire 1'  COMP_ID = 'Comp 1', FIRE_ID = 't squared'  LOCATION = 0.5, 0.5 IGNITION_CRITERION = 'TEMPERATURE', DEVC_ID = 'Targ 1', SETPOINT = 75 / 
&FIRE ID = 'Fire 2'  COMP_ID = 'Comp 1', FIRE_ID = 'Constant'  LOCATION = 4.5, 4.5 / 
&CHEM ID = 't squared' CARBON = 1 CHLORINE = 0 HYDROGEN = 4 NITROGEN = 0 OXYGEN = 0 HEAT_OF_COMBUSTION = 50000 RADIATIVE_FRACTION = 0.35 / 
&TABL ID = 't squared' LABELS = 'TIME', 'HRR' , 'HEIGHT' , 'AREA' , 'CO_YIELD' , 'SOOT_YIELD' , 'HCN_YIELD' , 'HCL_YIELD' , 'TRACE_YIELD'  /
&TABL ID = 't squared', DATA = 0, 0, 0, 0.001, 0.01, 0.01, 0, 0, 0 /
&TABL ID = 't squared', DATA = 30, 3, 0, 0.006883863, 0.01, 0.01, 0, 0, 0 /
&TABL ID = 't squared', DATA = 60, 12, 0, 0.02086797, 0.01, 0.01, 0, 0, 0 /
&TABL ID = 't squared', DATA = 90, 27, 0, 0.03992328, 0.01, 0.01, 0, 0, 0 /
&TABL ID = 't squared', DATA = 120, 48, 0, 0.06325986, 0.01, 0.01, 0, 0, 0 /
&TABL ID = 't squared', DATA = 150, 75, 0, 0.0904033, 0.01, 0.01, 0, 0, 0 /
&TABL ID = 't squared', DATA = 180, 108, 0, 0.1210248, 0.01, 0.01, 0, 0, 0 /
&TABL ID = 't squared', DATA = 210, 147, 0, 0.1548778, 0.01, 0.01, 0, 0, 0 /
&TABL ID = 't squared', DATA = 240, 192, 0, 0.191768, 0.01, 0.01, 0, 0, 0 /
&TABL ID = 't squared', DATA = 270, 243, 0, 0.2315369, 0.01, 0.01, 0, 0, 0 /
&TABL ID = 't squared', DATA = 300, 300, 0, 0.2740515, 0.01, 0.01, 0, 0, 0 /
&TABL ID = 't squared', DATA = 900, 300, 0, 0.2740515, 0.01, 0.01, 0, 0, 0 /
&CHEM ID = 'Constant' CARBON = 1 CHLORINE = 0 HYDROGEN = 4 NITROGEN = 0 OXYGEN = 0 HEAT_OF_COMBUSTION = 50000 RADIATIVE_FRACTION = 0.35 / 
&TABL ID = 'Constant' LABELS = 'TIME', 'HRR' , 'HEIGHT' , 'AREA' , 'CO_YIELD' , 'SOOT_YIELD' , 'HCN_YIELD' , 'HCL_YIELD' , 'TRACE_YIELD'  /
&TABL ID = 'Constant', DATA = 0, 50, 0, 0.1137982, 0, 0, 0, 0, 0 /
&TABL ID = 'Constant', DATA = 900, 50, 0, 0.1137982, 0, 0, 0, 0, 0 /
 
!! Devices
&DEVC ID = 'Targ 1' COMP_ID = 'Comp 1' LOCATION = 4.5, 4.5, 2.9 TYPE = 'PLATE' MATL_ID = 'Steel'  NORMAL = 0, 0, -1 TEMPERATURE_DEPTH = 0.00075 DEPTH_UNITS = 'M' /
```

**Comment by rpeacoc** [comment]
Open file in CEdit, select Fires Tab, and click on constant fire in the summary spreadsheet and the constant fire time-based data becomes the t-squared fire.

**Comment by rpeacoc** [comment]
Fixed in 4a8f735. Problem was a copy of the fire data in the wrong place
